### PR TITLE
Update metadata and images for embeds

### PR DESCRIPTION
**Pull request by Headscracher**
Continues #120

### CODE REVIEW

**Comment by Headscracher** [comment]
Wrong image is getting served. Will look later after that.

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -39,10 +40,25 @@ type AppPropsWithLayout = AppProps & {
...
+
 const SEO: DefaultSeoProps = {
   titleTemplate: '%s | Vegan Hacktivists',
   openGraph: {
     url: 'https://veganhacktivists.org',
+    description:
+      "We're a diverse community of passionate vegan activists from all around the world, volunteering our time and skills towards the animal protection movement. We build free technology and offer free services with the goal of reducing or stopping the mass amounts of suffering caused by factory farming.",
+    images: [
+      {
+        url: `${hostUrl}${logoBig.src}`,
```
Comment (trybick) [inline src/pages/_app.tsx:56]: Changed this URL to a full absolute URL (it was a relative URL before). Per this comment:
https://github.com/garmeeh/next-seo/issues/14#issuecomment-503715239

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -39,10 +40,25 @@ type AppPropsWithLayout = AppProps & {
   Component: NextPageWithLayout;
 };
 
+const hostUrl =
+  process.env.NODE_ENV === 'development'
+    ? 'localhost:3000'
+    : process.env.VERCEL_URL;
```
Comment (trybick) [inline src/pages/_app.tsx]: I'm assuming we use Vercel for deployment but would be good if someone can confirm.  If so, I believe this ENV variable could be a good way to get the current URL (see [here](https://stackoverflow.com/a/66011585/9830506) for more info on this). 

There are a couple other ways we go could with this like just hardcoding `https://veganhacktivists.org` or using a package called [next-absolute-url](https://www.npmjs.com/package/next-absolute-url).

Comment (trybick) [inline src/pages/_app.tsx]: Using VERCEL_URL here makes it so testing with gitpod doesn't quite work.  But this URL does work
https://3000-veganhackti-veganhackti-yb9g5zc3oib.ws-us80.gitpod.io/_next/static/media/VH_Logo_Crest_Tagline_Gray.98a43ba2.png

File: src/pages/_app.tsx [hunk 3]
```diff
@@ -39,10 +40,25 @@ type AppPropsWithLayout = AppProps & {
...
+
 const SEO: DefaultSeoProps = {
   titleTemplate: '%s | Vegan Hacktivists',
   openGraph: {
     url: 'https://veganhacktivists.org',
+    description:
+      "We're a diverse community of passionate vegan activists from all around the world, volunteering our time and skills towards the animal protection movement. We build free technology and offer free services with the goal of reducing or stopping the mass amounts of suffering caused by factory farming.",
+    images: [
+      {
+        url: `${hostUrl!}${logoBig.src}`,
```
Comment (trybick) [inline src/pages/_app.tsx]: Changed this URL to a full absolute URL (it was a relative URL before). Per this comment:
https://github.com/garmeeh/next-seo/issues/14#issuecomment-503715239

File: src/pages/_app.tsx (hunk 2 above)
Comment (Headscracher) [reply src/pages/_app.tsx]: We're using digitalocean app-platform. So yeah just use veganhacktivists.org hardcoded

**Pull request merged**